Still a warning present:
```
[ 95%] Linking CXX executable scope
/home/ksmith/paass/Acquisition/MCA/source/HisDrr.cpp: In constructor ‘HisDrr::HisDrr(const string&, const string&, const string&)’:
/home/ksmith/paass/Acquisition/MCA/source/HisDrr.cpp:162:29: warning: ‘totLength’ may be used uninitialized in this function [-Wmaybe-uninitialized]
     head.nHWords = totLength;
                             ^
```

--- a/Acquisition/MCA/source/HisDrr.cpp
+++ b/Acquisition/MCA/source/HisDrr.cpp
@@ -150,7 +150,7 @@ HisDrr::HisDrr(const string &drr, const string &his, const string &input) {
 
     // Using information from drrData drr header is created
     DrrHeader head;
-    int totLength;
+    int totLength = 0;
     for (unsigned int i = 0; i < drrData.size(); ++i) 
         totLength += (drrData[i].scaled[0]+drrData[i].scaled[1])*drrData[i].halfWords; 
     // Magic words (whatever they do...)

